stable/database/files/readinessprobe

```diff
@@ -46,9 +49,10 @@ pgrep -xa nuodb | (
         else
             # argv does not contain process info; use 'nuocmd show domain' to
             # obtain start ID and 'nuocmd check process' to check process
-            # health; this does not scale well as the number of processes
-            # becomes very large, but we have no choice
-            start_id="$(nuocmd show domain --process-format "==={hostname}:{pid} {start_id}" | sed -n "s/ *===$(hostname):$pid //p")"
+            # health; if multiple start IDs are returned, check the largest
+            # one, which has to be the current one; this does not scale well as
+            # the number of processes becomes very large, but we have no choice
+            start_id="$(nuocmd show domain --process-format "==={hostname}:{pid} {start_id}" | sed -n "s/ *===$(hostname):$pid //p" | sort -n | tail)"
```

Comment:
If you want to strip out all the stuff why not just print the start_id by itself and get rid of the sed?

Also, don't you want *tail -n1* to get just the last ID?

Reply:
> Also, don't you want tail -n1 to get just the last ID?

Whoops! Of couse, it should be `tail -n1`.

> If you want to strip out all the stuff why not just print the start_id by itself and get rid of the sed?

Do you mean just change the format string? The `sed` is filtering out all other processes. We only want the processes that are local to this host and that have the matching pid.

Reply:
Hm still getting used to this github interface.  Anyway I see WRT the sed, looks good.